build.gradle

```diff
@@ -44,27 +47,32 @@ run {
 // ### Checkstyle plugin settings
 checkstyle {
     // Latest version at https://github.com/checkstyle/checkstyle/tags
-    toolVersion = '10.12.1'
+    toolVersion = '10.23.1'
+
     // Keep the below file updated along with subsequent versions of Checkstyle (make sure to choose
     // the tag matching the toolVersion above):
     // https://github.com/checkstyle/checkstyle/blob/checkstyle-10.3.2/src/main/resources/google_checks.xml
+
```

Comment:
Nit: prefer not to add this line break, unless you // it. I think it's easier for someone to overlook the instructions below it unless it's all merged together, or the line breaks are clearly indicating a continuation of the comment.